### PR TITLE
docs: add info on new api lb routing modes

### PR DESCRIPTION
**Pull request by Crispy1975**
## What kind of change does this PR introduce?
Docs update for incoming API load balancer routing mode.

https://docs-ou07fwklb-supabase.vercel.app/docs/guides/platform/read-replicas#experimental-routing

### CODE REVIEW

File: apps/docs/content/guides/platform/read-replicas.mdx [hunk 1]
```diff
@@ -102,6 +102,15 @@
 
 If you remove all Read Replicas from your project, the load balancer and its endpoint are removed as well. Make sure to redirect requests back to your Primary database before removal.
 
+#### Experimental Routing
+
+The API load balancer offers an experimental routing mode that build on top of the existing funtionality. It is now possible to use your load balancer endpoint for all Supabase related traffic. The following list summarises the features that are now accessible.
```
[the remark this answers is not in the document]
Comment (Crispy1975) [reply apps/docs/content/guides/platform/read-replicas.mdx]: Fixed

File: apps/docs/content/guides/platform/read-replicas.mdx [hunk 2]
```diff
@@ -102,6 +102,15 @@
...
+
+- Auth requests are sent to the Primary database even when using the load balancer endpoint.
+- Routing of Edge Functions, Storage, and Realtime requests to the appropriate backend Supabase service.
+
+The experimental features listed can be enabled by sending the header `sb-lb-routing-mode: alpha-all-services` along with any requests. This gives the option to opt-in and try the new features out, but still have the peace of mind that the original behaviour is there should it be needed. Please let us know if you have any feedback by [contacting us](https://supabase.com/support).
```
[the remark this answers is not in the document]
Comment (Crispy1975) [reply apps/docs/content/guides/platform/read-replicas.mdx]: Fixed.

File: apps/docs/content/guides/platform/read-replicas.mdx [hunk 3]
```diff
@@ -102,6 +102,15 @@ To call a read-only Postgres function on Read Replicas through the REST API, use
 
 If you remove all Read Replicas from your project, the load balancer and its endpoint are removed as well. Make sure to redirect requests back to your Primary database before removal.
 
+#### Experimental Routing
+
+The API load balancer offers an experimental routing mode that builds on top of the existing funtionality. It is now possible to use your load balancer endpoint for all Supabase related traffic. The following list summarizes the features that are now accessible.
+
+- Auth requests are sent to the Primary database even when using the load balancer endpoint.
+- Routing of Edge Functions, Storage, and Realtime requests go to the appropriate backend Supabase service.
```
Comment (darora) [inline apps/docs/content/guides/platform/read-replicas.mdx:110]: While internally this is a routing problem in terms of implementation, from the user's perspective it's really more of a "The LB endpoint now supports usage of additional services (Auth, Storage, Realtime, Edge Functions)."

Can add a note that "Due to the requirements of the Auth service, all Auth requests are handled by the Primary, even when sent over the LB endpoint, similar to how non-Read requests for the Data API (Postgrest) are exclusively handled by the Primary."

Comment (Crispy1975) [reply apps/docs/content/guides/platform/read-replicas.mdx]: Updated.

**Pull request closed without merging**